src/app/accessRequests/rejectSubServiceRequest.js

```diff
@@ -1,78 +1,82 @@
-const {updateServiceRequest} = require('../requestService/utils');
+const { updateServiceRequest } = require('../requestService/utils');
 const Account = require('./../../infrastructure/account');
 const { isServiceEmailNotificationAllowed } = require('../../../src/infrastructure/applications');
-const { getSubServiceRequestVieModel, getAndMapServiceRequest, getRoleAndServiceNames, generateFlashMessages } = require('./utils');
+const {
+  getSubServiceRequestVieModel,
+  getAndMapServiceRequest,
+  getRoleAndServiceNames,
+  generateFlashMessages,
+} = require('./utils');
 const logger = require('./../../infrastructure/logger');
 const config = require('./../../infrastructure/config');
 const NotificationClient = require('login.dfe.notifications.client');
 
-
 const validate = async (req) => {
-  const buildmodel = await getAndMapServiceRequest(req.params.rid); 
+  const buildmodel = await getAndMapServiceRequest(req.params.rid);
   const viewModel = await getSubServiceRequestVieModel(buildmodel, req.id, req);
   viewModel.selectedResponse = req.body.selectedResponse;
-  if( req.session.roleIds !== undefined)
-  {
-    if( req.session.roleIds !== viewModel.role_ids)
-    {
-    viewModel.role_ids = req.session.roleIds;
-     let submodel =  await getRoleAndServiceNames(viewModel, req.params.rid, req);
-     viewModel.roles = submodel.roles.filter(x => x !== undefined);
-     //req.session.roleIds = undefined;
-     req.session.roles = viewModel.roles;
-    }else{
-     // req.session.roleIds = undefined;
+  if (req.session.roleIds !== undefined) {
+    if (req.session.roleIds !== viewModel.role_ids) {
+      viewModel.role_ids = req.session.roleIds;
+      let submodel = await getRoleAndServiceNames(viewModel, req.params.rid, req);
+      viewModel.roles = submodel.roles.filter((x) => x !== undefined);
+      //req.session.roleIds = undefined;
+      req.session.roles = viewModel.roles;
+    } else {
+      // req.session.roleIds = undefined;
```

Comment:
can we please remove the commented out lines?

Reply:
done